game/test/graphic/DungeonCameraTest.java

```diff
@@ -1,135 +1,230 @@
-     }
-
-     @Test
-     public void test_getFollowedObject_noFollow() {
-         assertNull(cam_noFollow.getFollowedObject());
-         Mockito.verify(cam_noFollow).getFollowedObject();
-         Mockito.verifyNoMoreInteractions(cam_noFollow, pos_noFollow, frustum_noFollow);
-     }
-
-     @Test
-     public void test_setFocusPoint() {
-         cam.setFocusPoint(new Point(2, 2));
-         assertNull(cam.getFollowedObject());
-         Mockito.verify(cam).setFocusPoint(any());
-         Mockito.verify(cam).getFollowedObject();
-         Mockito.verifyNoMoreInteractions(cam, pos, frustum, entity);
-     }
-
-     @Test
-     public void test_setFocusPoint_noFollow() {
-         cam_noFollow.setFocusPoint(new Point(2, 2));
-         assertNull(cam_noFollow.getFollowedObject());
-         Mockito.verify(cam_noFollow).setFocusPoint(any());
-         Mockito.verify(cam_noFollow).getFollowedObject();
-         Mockito.verifyNoMoreInteractions(cam_noFollow, pos_noFollow, frustum_noFollow);
-     }
-
-     @Test
-     public void test_isPointInFrustum() {
-         cam.isPointInFrustum(2, 2);
-         Mockito.verify(cam).isPointInFrustum(2, 2);
-         Mockito.verify(frustum).boundsInFrustum(any());
-         Mockito.verifyNoMoreInteractions(cam, pos, frustum, entity);
-     }
-
-     @Test
-     public void test_isPointInFrustum_noFollow() {
-         cam_noFollow.isPointInFrustum(2, 2);
-         Mockito.verify(cam_noFollow).isPointInFrustum(2, 2);
-         Mockito.verify(frustum_noFollow).boundsInFrustum(any());
-         Mockito.verifyNoMoreInteractions(cam_noFollow, pos_noFollow, frustum_noFollow);
-     }
-
-     @Test
-     public void test_getFrustum() {
-         Mockito.verifyNoMoreInteractions(cam, pos, frustum, entity);
-     }
-
-     @Test
-     public void test_getFrustum_noFollow() {
-         Mockito.verifyNoMoreInteractions(cam_noFollow, pos_noFollow, frustum_noFollow);
-     }
-    */
+
+    @BeforeClass
+    public static void setUpGdx() {
+        GdxNativesLoader.load(); // load natives for headless testing
+    }
+
+    @Test
+    public void test_update() {
+        // Prepare
+        Entity entity = new Entity();
+        PositionComponent positionComponent = new PositionComponent(entity, new Point(2, 2));
+        DungeonCamera camera =
+                new DungeonCamera(
+                        positionComponent, Constants.VIEWPORT_WIDTH, Constants.VIEWPORT_HEIGHT);
+
+        // Testing
+        assertEquals(
+                "Pos should be (0, 0, 0) initially.",
+                camera.position,
+                new Vector3(0, 0, 0)); // Camera is initially at (0, 0, 0)
+        camera.update();
+        assertEquals(
+                "Pos should be (2, 2, 0) after update.",
+                camera.position,
+                new Vector3(2, 2, 0)); // Because it follows the positionComponent
+        positionComponent.setPosition(new Point(15, 23));
```

Comment:
Sollte hier nicht eher 
```suggestion
        entity.setPosition(new Point(15, 23));
```
stehen, damit wir sicherstellen, dass auch der unterliegenden Entity gefolgt wird?

Reply:
Ja, verstehe deinen Gedanken, kann ich auch ändern, allerdings wäre das ja für diesen Test nicht relevant, da sich die Camera am `PositionComponent` orientiert und nicht am Entity.

Reply:
Ich sehe auch gerade, dass das gar nicht geht, da `Entity#setPosition` nicht existiert 🤷‍♂️